proxy.js

```diff
@@ -171,4 +171,4 @@
 
   scope.Proxy['revocable'] = scope.Proxy.revocable;
   scope['Proxy'] = scope.Proxy;
-})(typeof module !== 'undefined' && module['exports'] ? global : window);
+})(typeof process !== "undefined" && {}.toString.call(process) === "[object process]" ? global : typeof WorkerGlobalScope !== 'undefined' && self instanceof WorkerGlobalScope ? self : window);
```

Comment:
So then what you want is-

```js
typeof process !== 'undefined' && {}.toString.call(process) == '[object process]' ? global : self;
```

Does SystemJS not have `global`? Why would it have a false positive on `module.exports` - surely SystemJS is designed to import anything, including things exported via `module.exports`?